Frameworks/Foundation/NSHTTPURLResponse.mm

```diff
@@ -35,6 +35,24 @@ @implementation NSHTTPURLResponse {
     StrongId<NSString> _HTTPVersion;
 }
 
+NSMutableDictionary* _constructCaseInsensitiveDictionary(NSDictionary* dict) {
+    CFDictionaryKeyCallBacks caseInsensitiveKeyChecker = kCFTypeDictionaryKeyCallBacks;
```

Comment:
FWIW github mistakenly aimed this at the line below the line I was intending/

---
In reply to: [72358654](https://github.com/Microsoft/WinObjC/pull/673#discussion_r72358654) [](ancestors = 72358654)